### PR TITLE
Dlp: Regeneration and hand edits

### PR DESCRIPTION
**Pull request by landrito**
I left nox.py intact.

Generated with config edits in https://github.com/googleapis/googleapis/pull/430

### CODE REVIEW

File: dlp/README.rst [hunk 1]
```diff
@@ -59,6 +59,26 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install gapic-google-cloud-dlp-v2beta1
 
+Preview
+~~~~~~~
+
+DlpServiceClient
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: py
```
Comment (lukesneeringer) [inline dlp/README.rst]: What is the deal with this? This is hideous looking. I assume it was auto-genned?

File: dlp/google/cloud/proto/privacy/dlp/v2beta1/dlp_pb2_grpc.py [hunk 2]
```diff
@@ -62,23 +62,24 @@ class DlpServiceServicer(object):
   """
 
   def InspectContent(self, request, context):
-    """Find potentially sensitive info in a list of strings.
+    """Finds potentially sensitive info in a list of strings.
```
Comment (lukesneeringer) [inline dlp/google/cloud/proto/privacy/dlp/v2beta1/dlp_pb2_grpc.py]: These managed to get worse. Was this just an upstream change to the comments that created this docstring?

File: dlp/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+googleapis-common-protos>=1.5.2, <2.0dev
```
Comment (lukesneeringer) [inline dlp/requirements.txt]: We should delete this file.

File: dlp/README.rst (hunk 1 above)
Comment (landrito) [inline dlp/README.rst]: Correct this is auto generated.

File: dlp/google/cloud/proto/privacy/dlp/v2beta1/dlp_pb2_grpc.py (hunk 2 above)
Comment (landrito) [inline dlp/google/cloud/proto/privacy/dlp/v2beta1/dlp_pb2_grpc.py]: Correct. https://github.com/googleapis/googleapis/commit/699916405828ab150123f41c341ac35fabd56a9d

File: dlp/requirements.txt (hunk 3 above)
Comment (landrito) [inline dlp/requirements.txt]: Done.

File: dlp/README.rst (hunk 1 above)
Comment (lukesneeringer) [inline dlp/README.rst]: Okay, thanks.

Comment (landrito) [inline dlp/README.rst]: I made this PR: https://github.com/googleapis/toolkit/pull/1494, which should make this look slightly better.

Comment (landrito) [inline dlp/README.rst]: As well as https://github.com/googleapis/toolkit/pull/1493 to fix the header lines.

**Pull request merged**